modules/assim.sequential/R/sda.enkf.R

```diff
@@ -73,7 +84,9 @@ sda.enkf <- function(settings, IC, prior, obs.mean, obs.cov, variables,
     ensemble.id <- -1
   }
   
-  ## model-specific functions
+  ###-------------------------------------------------------------------###
+  ### get model specific functions                                      ###
+  ###-------------------------------------------------------------------### 
   do.call("require",list(paste0("PEcAn.",model)))
   my.write.config <- paste("write.config.",model,sep="")
```

Comment:
meant to write my.write.restart, which seems to exist below but I didn't see where it was defined.